tests/storage/cross_cluster_live_migration/test_cclm.py

```
@@ -18,21 +34,30 @@
 
 
 @pytest.mark.parametrize(
-    "vms_for_cclm",
+    "remote_cluster_source_storage_class, local_cluster_target_storage_class, vms_for_cclm",
```

Ruff (0.15.1)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)